protos/format.proto

```diff
@@ -58,6 +63,9 @@ message Manifest {
 
   // Schema metadata.
   map<string, bytes> metadata = 5;
+
+  // If presented, the file position of the index metadata.
+  optional uint64 index_section = 6;
```

Comment:
So proto2 only?

Reply:
TIL that `proto3` treats optional as explicitly presence https://protobuf.dev/programming-guides/field_presence/